### PR TITLE
Merge two translation contributing documents

### PR DESCRIPTION
**Pull request by lunny**
This PR also adjusts the weights.

### CODE REVIEW

**Comment by wolfogre** [comment]
Sorry I don't get it. Merge two translation contributing documents to what? The two documents are both deleted.

File: docs/content/doc/contributing/localization.de-de.md [hunk 1]
```diff
@@ -1,17 +1,39 @@
 ---
-date: "2021-01-22T00:00:00+02:00"
-title: "Übersetzungs Richtlinien"
```
Comment (wolfogre) [inline docs/content/doc/contributing/localization.de-de.md]: Sure? The file name is localization.de-de.md

File: docs/content/doc/contributing/localization.en-us.md [hunk 2]
```diff
@@ -31,3 +31,7 @@ After a translation has been accepted, it will be reflected in the main reposito
 At the time of writing, this means that a changed translation may not appear until the following Gitea release.
 
 If you use a bleeding edge build, it should appear as soon as you update after the change is synced.
+
+# How to Contribute
+
+Different Languages have different translation guidelines. Please go to their own page to get more informations.
```
Comment (delvh) [inline docs/content/doc/contributing/localization.en-us.md]:
```suggestion
Different Languages have different translation guidelines. Please visit the respective page for more information.
```

File: docs/content/doc/contributing/localization.de-de.md (hunk 1 above)
Comment (delvh) [reply docs/content/doc/contributing/localization.de-de.md]: Yup, and suddenly the German file is partly written in English?
I don't quite see how that is an improvement yet…

**Comment by lunny** [comment]
@wolfogre @delvh please review again

**Review by delvh — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**